### DISCUSSION TITLE
[Feature Request]: Ability to reset site PHP version once it is set

### DISCUSSION THREAD

**Discussion by iruzevic** [beyondcode/herd-community#348] [Ideas]
### Feature Description

Site PHP version will never reset to global one (mac toolbar) if it is once set using sites settings. It is awesome that we can change individual sites PHP version but it would be awesome if we could "reset" this setting once it is set to follow the global settings.

Video for explanation:

https://github.com/beyondcode/herd-community/assets/23283324/f99f5d8c-f4e3-4277-a837-46c6e6760ede

**Comment by neurotools** [comment]
It will be really helpful!!

Now you can "unisolate" the site from the command line with:

`herd unisolate`

But there's no option in the GUI to set the PHP version to "herd default"
